tests/testHSCColorterms.py

```diff
@@ -0,0 +1,97 @@
+        self.assertEqual(ctg.primary, "g")
+        self.assertEqual(ctg.secondary, "g")
+        self.assertEqual(ctg.c0, 0.0)
+        self.assertEqual(ctg.c1, 0.0)
+        self.assertEqual(ctg.c2, 0.0)
+
+    def testSDSSColorterms(self):
+        """Test that the colorterm libraries are formatted correctly"""
+        sdssBands = ["g","r","i","z","y"]
+        hscBands = ["g","r","i","i2","z","y","N816","N921"]
+        for band in hscBands:
+            ct = self.photoCalConf.colorterms.getColorterm(band, photoCatName="sdss") # exact match
+            self.assertTrue(ct.primary in sdssBands)
+            self.assertTrue(ct.secondary in sdssBands)
+            self.assertTrue(isinstance(ct.c0, numbers.Number))
+            self.assertTrue(isinstance(ct.c1, numbers.Number))
+            self.assertTrue(isinstance(ct.c2, numbers.Number))
+
+    def testHSCColortermG(self):
+        """Test that the 'N816' colorterm matches the values in config/colorterms.py""" 
+        ctn816 = self.photoCalConf.colorterms.getColorterm("N816", photoCatName="sdss") # exact match
+        self.assertEqual(ctn816.primary, "i")
+        self.assertEqual(ctn816.secondary, "z")
+        self.assertEqual(ctn816.c0, 0.00927133)
+        self.assertEqual(ctn816.c1, -0.63558358)
+        self.assertEqual(ctn816.c2, -0.05474862)
+
+    def testPS1Colorterms(self):
+        """Test that the colorterm libraries are formatted correctly"""
+        ps1Bands = ["g","r","i","z","y"]
+        hscBands = ["g","r","i","i2","z","y","N816","N921"]
+        for band in hscBands:
+            ct = self.photoCalConf.colorterms.getColorterm(band, photoCatName="ps1") # exact match
+            self.assertTrue(ct.primary in ps1Bands)
+            self.assertTrue(ct.secondary in ps1Bands)
+            self.assertTrue(isinstance(ct.c0, numbers.Number))
+            self.assertTrue(isinstance(ct.c1, numbers.Number))
+            self.assertTrue(isinstance(ct.c2, numbers.Number))
+
+    def testHSCColortermG(self):
+        """Test that the 'i2' colorterm matches the values in config/colorterms.py""" 
+        cti2 = self.photoCalConf.colorterms.getColorterm("i2", photoCatName="ps1") # exact match
+        self.assertEqual(cti2.primary, "i")
+        self.assertEqual(cti2.secondary, "z")
+        self.assertEqual(cti2.c0, 0.00180361)
+        self.assertEqual(cti2.c1, -0.18483562)
+        self.assertEqual(cti2.c2, -0.02675511)
+
+#-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
+
+
+def suite():
+    """Returns a suite containing all the test cases in this module."""
+
+    utilsTests.init()
+
+    suites = []
+    suites += unittest.makeSuite(ColortermOverrideTestCase)
+    suites += unittest.makeSuite(utilsTests.MemoryTestCase)
+    return unittest.TestSuite(suites)
+
+
+def run(shouldExit=False):
+    """Run the tests"""
+    utilsTests.run(suite(), shouldExit)
```

Comment:
@timj is going to be along in a moment to tell you that this is deprecated and you should be following https://developer.lsst.io/coding/python_testing.html instead!